libafl/src/events/tcp.rs

```diff
@@ -75,6 +77,8 @@ where
     phantom: PhantomData<I>,
 }
 
+const UNDEFINED_CLIENT_ID: ClientId = ClientId(0xffffffff);
```

Comment:
Should we have an Option instead of this?
Else we should make this a global ClientId feature (and/or use an Enum)

Reply:
an option will be larger than u32, the point is that is used in the request to ask for an id assignment

Reply:
Option<NonZeroU32> would work